text/0000-template-changes.md

```diff
@@ -0,0 +1,61 @@
+# Meta
+[meta]: #meta
+- Name: RFC template changes
+- Start Date: 2020-02-03
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+As a response to the process of approving [RFC 0013](0013-app-layer-metadata-source.md) which ended up requiring spec changes, this RFC proposes amending the template to include an optional (non-binding) section to document spec changes that fall out of an RFC.
+
+# Motivation
+[motivation]: #motivation
+
+Improve the process of moving RFCs into spec and implementation.
+
+# What it is
+[what-it-is]: #what-it-is
+
+[0000-template.md](../0000-template.md) gets a new section:
+
+```diff
+
+# Prior Art
+[prior-art]: #prior-art
+
+Discuss prior art, both the good and bad.
+
++
++ # Spec. Changes (OPTIONAL)
++ [spec-changes]: #spec-changes
++
++ Does this RFC entail any proposed changes to the specifications (either buildpack or platform) or extensions?
++ This section is not intended to be binding, but as discussion of an RFC unfolds, if spec changes are necessary, they should be documented here.
++
```

Comment:
It would be nice to provide some examples here to help folks who are less familiar with the spec start to consider this question: new lifecycle flags, new buildpack.toml fields, new fields in the buildpackage label etc.

Reply:
> It would be nice to provide some examples here to help folks who are less familiar with the spec start to consider this question: new lifecycle flags, new buildpack.toml fields, new fields in the buildpackage label etc.

Agree that examples would be helpful, will add. However, I don't think me message here is that the onus is necessarily on the RFC opener, at least initially, to add to this section. My thinking was that if an RFC evolved to a place where spec changes where necessary, a reviewer should call that out and the author would add it on when they received that feedback.

Reply:
Can we get the examples added here?